website/other/bezier-rs-demos/wasm/src/bezier.rs

```diff
@@ -0,0 +1,256 @@
+use crate::svg_drawing::*;
+use bezier_rs::{ArcStrategy, ArcsOptions, Bezier, ProjectionOptions, ToSVGOptions};
+use glam::DVec2;
+use serde::{Deserialize, Serialize};
+use wasm_bindgen::prelude::*;
+
+#[derive(Serialize, Deserialize)]
+struct CircleSector {
+	center: Point,
+	radius: f64,
+	#[serde(rename = "startAngle")]
+	start_angle: f64,
+	#[serde(rename = "endAngle")]
+	end_angle: f64,
+}
```

Comment:
Perhaps a clearer name for this would be `Arc`?

Reply:
I'm happy with either. I agree "Arc" might be the more official name, but I do think "circle sector" may be more specific and less overloaded. To me at least, I get a more intuitive understanding from the current name. I don't mind either way.